client/src/pages/CalendarPage.js

```diff
@@ -251,18 +250,25 @@ export default function CalendarPage() {
                 return
             };
         });
+        if (moment(selectedDay).format("YYYY-DDD") === moment(new Date()).format("YYYY-DDD")){  
```

Comment:
The "YYYY-DDD" format is full year with the number of the day in the year: 1 to 365